eng/mgmt/changelog/src/main/java/com/azure/resourcemanager/tools/changelog/changelog/ChangeLog.java

```diff
@@ -97,6 +100,26 @@ public boolean isClassLevelChanged() {
 
     protected void calcChangeLog() {
         calcChangeLogForClass();
+        // When a base method overridden by child class got changed, two identical change logs will appear for this child class
+        deduplicateChangeLog();
+    }
+
+    private void deduplicateChangeLog() {
+        deduplicate(this.breakingChange);
+        deduplicate(this.newFeature);
+    }
+
+    private void deduplicate(List<String> changeList) {
+        Set<String> changeSet = new HashSet<>();
+        Iterator<String> iterator = changeList.iterator();
+        while (iterator.hasNext()) {
+            String change = iterator.next();
+            if (changeSet.contains(change)) {
+                iterator.remove();
```

Comment:
Do we have test to cover this case?

Reply:
Tested locally. Adding unit test case is a little bit tricky, since it involves comparing two jar files.